alpha/action/diff_test.go

```diff
@@ -129,6 +133,7 @@ func TestDiff(t *testing.T) {
 						},
 					},
 				},
+				IncludeAdditively: true,
```

Comment:
Do we want to add some test cases where IncludeAdditively is false for explicit output comparison?

Reply:
The tests I added elsewhere cover this case. These are mostly to ensure input is parsed and output is rendered correctly with real files.